src/test/java/de/zalando/aruha/nakadi/security/AuthenticationTest.java

```diff
@@ -165,9 +173,19 @@ public void setup() {
         mockMvc = MockMvcBuilders.webAppContextSetup(applicationContext).addFilters(springSecurityFilterChain).build();
     }
 
-    protected MockHttpServletRequestBuilder addTokenHeader(final MockHttpServletRequestBuilder builder,
-            final String token) {
-        return builder.header("Authorization", "Bearer " + token);
+    @Test
+    public void testPublicEndpoints() {
+        endpointsWithNoScope.forEach(this::checkOkResponse);
+    }
+
+    private void checkOkResponse(Endpoint endpoint) {
+        try {
+            mockMvc.perform(endpoint.toRequestBuilder()).andExpect(status().isOk());
+        } catch (Exception e) {
+            throw new AssertionError("Exception occurred while calling endpoint: " + endpoint, e);
+        } catch (AssertionError e) {
+            throw e;
```

Comment:
Acceptance tests fail in travis https://travis-ci.org/zalando/nakadi/jobs/122999756
It timed out at the place where it waits for nakadi-container. Maybe be it is somehow connected with changes you did in Makefile or gradle. On my PC the build and tests work fine for your branch. I think you can first try to relaunch the build on travis, maybe that was some travis problem.